routes/oauth.js

```diff
@@ -42,7 +42,6 @@ router.get("/init", jsonParser, (req, res) => {
 
 router.get("/confirmed", jsonParser, async (req, res) => {
   if (!req.session.state) {
-    // TODO: Needs better error message
```

Comment:
nit: we may want to add this TODO back, or something.
The current error message is near useless and could probably stand a bit of improved copy. (although I'm happy to submit a new issue for that as well, and not block this PR)